src/style.css

```diff
@@ -0,0 +1,71 @@
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Book.ttf");
+  font-weight: bold;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
+
+.container {
+  margin: 20px 0;
+}
+
+.searchbar {
+  position: relative;
```

Comment:
remove position relative here, you don't need it

position relative should be on the parent of the element with position absolute